3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -494,11 +494,24 @@ object EnrichmentManager {
       case None => None.success
     }
 
+    // Fetch IAB enrichment context
+    val iabContext = registry.getIabEnrichment match {
+      case Some(iab) =>
+        iab
+          .getIabContext(Option(event.useragent),
+                         Option(event.user_ipaddress),
+                         Option(event.derived_tstamp).map(EventEnrichments.fromTimestamp))
+          .map(_.some)
+      case None => None.success
+    }
```

Comment:
One thing I missed before, at this point `event.user_ipaddress` may have been modified by anon ip, no?

Reply:
Good point, I'll move this step to be executed just after the IP lookups enrichment - that shouldn't be affected by anonymization